sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1205,6 +1239,7 @@ object RapidsConf {
         s"Executors that don't heartbeat within this timeout will be considered stale. " +
         s"This timeout must be higher than the value for " +
         s"${SHUFFLE_TRANSPORT_EARLY_START_HEARTBEAT_INTERVAL.key}")
+      .startupOnly()
       .integerConf
       .createWithDefault(10000)
 
```

Comment:
all the shuffle and ucx ones below are `startupOnly` as well. The only shuffle config that isn't is: `spark.rapids.shuffle.enabled`, this one you can change at runtime.

Reply:
Updated shuffle and ucx configs as startupOnly.

Reply:
because I tested the auto mount one and if it wasn't there initially on startup things would fail.   You can turn it on and off after startup with it enabled.  I think the init code or something only did it at startup, you can see the description that I documented it there as a requirement.

Reply:
I guess perhaps the other Alluxio ones aren't explicitly required at startup, although the regex ones requires you to have mounted the buckets already or I guess if you can mount run time it would work so I'm ok with those

Reply:
With the fixing, auto mount should be changable at runtime.

Reply:
@tgravescs Is it ok to resolve this conversation?